src/main/java/reposense/authorship/FileInfoAnalyzer.java

```diff
@@ -135,7 +137,7 @@ private static FileResult generateBinaryFileResult(RepoConfiguration config, Fil
 
     /**
      * Sets the {@code Author} and {@code Date} for each line in {@code fileInfo} based on the git blame analysis
-     * on the file.
+     * on the file and {@code config}.
```

Comment:
This comment does not really make sense. We are not doing a git blame analysis on config, consider rephrasing this sentence?